docs/guide/dev/integration/vue/README.md

```diff
@@ -329,6 +329,32 @@ Corresponds to the {@linkapi CKEDITOR.editor#event-change `change`} editor event
 <ckeditor @input="onEditorInput"></ckeditor>
 ```
 
+### `namespaceloaded`
+
+Fired once {@linkapi CKEDITOR} namespace is loaded on the page. This event is called only one time for every editor instance loaded on the page and only when using [`editor-url`](#editor-url) directive with valid CKEditor 4 CDN URL. Allows to modify global editor namespace.
+
+```html
+<ckeditor @namespaceloaded="onNamespaceLoaded" :config="editorConfig"></ckeditor>
+```
+
+```js
+new Vue( {
+	el: '#app',
+	data: function() {
+		editorConfig: {
+			extraPlugins: 'placeholder'
+		}
+	},
+	methods: {
+		onNamespaceLoaded( CKEDITOR ) {
+			// Add external `placeholder` plugin which will be available for every
+			// editor instance on the page.
+			CKEDITOR.plugins.addExternal( '/path/to/the/placeholder/plugin', 'plugin.js' );
```

Comment:
[`CKEDITOR.plugins.addExternal()`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins.html#method-addExternal) requires a plugin name as the first parameter.